docs/standard/modern-web-apps-azure-architecture/azure-hosting-recommendations-for-asp-net-web-apps.md

```diff
@@ -39,9 +39,50 @@ App Service Web Apps offers a fully managed platform optimized for hosting web a
+
+There are a few scenarios where Web Apps for Containers make the most sense. If you have existing apps that you're able to containerize, whether in Windows or Linux containers, you'll be able to host these easily using this toolset. Simply publish your container and then configure Web Apps for Containers to pull the latest version of that image from your registry of choice. This is a "lift and shift" approach to migrating from classic app hosting models to a cloud-optimized model.
+
+![](./media/image1-8.png)
+
+This approach also works well if your development team is able to move to a container-based development process. The "inner loop" of developing apps with containers includes building the app with containers. Changes made to the code as well as to container configuration are pushed to source control, and an automated build is responsible for publishing new container images to a registry like Docker Hub or Azure Container Registry. These images are then used as the basis for additional development, as well as for deployments to production, as shown in the following diagram:
+
+![](./media/image1-7.png)
+
+Developing with containers offers numerous advantages, especially when containers are used in production. The same container configuration is used to host the app in each environment in which it runs, from local dev machine to build and test systems to production. This greatly reduces the likelihood of defects resulting from differences in machine configuration or software versions. Developers can also use whatever tools they're most productive with, including operating system, since containers can run on any OS. In some cases, distributed applications involving many containers may be very resource intensive to run on a single dev machine. In this scenario, developers can use Azure Dev Spaces to rapidly build container-based apps composed of many different containers in the cloud.
+
+Azure Dev Spaces helps teams to focus on the development and rapid iteration of their microservice application by allowing teams to work directly with their entire microservices architecture or application running in AKS. Azure Dev Spaces also provides a way to independently update portions of your microservices architecture in isolation without affecting the rest of the AKS cluster or other developers.
```

Comment:
Azure Dev Spaces is an AKS only feature. We shouldn't be writing this here in "App service web apps for containers". IMO, It's best to move it to the AKS section and keep it really brief. For Web Apps, we shouldn't be recommending AKS or Dev Spaces but a brief mention is OK. AKS is covered in-depth in Microservices guidance.

Reply:
Moved to AKS section.